test/includes/setup.sh

```diff
@@ -23,7 +23,8 @@ ensure_import_testimage() {
                 exit 1
             fi
 
-            deps/import-busybox --alias testimage
+            project=$(lxc project list | grep '(current)' | cut -d' ' -f2)
+            deps/import-busybox --alias testimage --project "$project"
```

Comment:
Can you move this change to its own commit as it's effectively a generic fix to `ensure_import_testimage` to respect projects.